test/index-intervals.spec.js

```diff
@@ -1,100 +1,142 @@
-
-    it('writes and reads with yearly indexes', function() {
-        return check('year', '2014');
-    });
+        }
 
-    it('writes and reads with one index', function() {
-        return check('none', '');
-    });
+        it('writes and reads with daily indices', function() {
+            return check('day');
+        });
 
-    describe('errors', function() {
-        function check_write_fail(promise, message) {
-            return check_no_write(expect_to_fail(promise, message));
-        }
-        it('bogus interval', function() {
-            var message = 'invalid interval: bananas; accepted intervals are "day", "week", "month" "year", and "none"';
-            var bad_opts = {indexInterval: 'bananas'};
-            return Promise.all([
-                expect_to_fail(test_utils.read(bad_opts), message),
-                check_write_fail(test_utils.write(points, bad_opts), message)
-            ]);
+        it('writes and reads with weekly indexes', function() {
+            return check('week');
         });
 
-        it('star in middle of write index', function() {
-            var index = 's*tar';
-            var message = 'cannot write to index pattern: ' + index;
-            return check_write_fail(test_utils.write([{}], {index: index}), message);
+        it('writes and reads with monthly indexes', function() {
+            return check('month');
         });
 
-        it('indexInterval and no star', function() {
-            var message = 'with indexInterval, index must end in *';
-            var index = 'no_star';
-            var bad_opts = {index: index, indexInterval: 'day'};
-            return Promise.all([
-                expect_to_fail(test_utils.read(bad_opts), message),
-                check_write_fail(test_utils.write([{}], bad_opts), message)
-            ]);
+        it('writes and reads with yearly indexes', function() {
+            return check('year');
         });
 
-        it('star in write index for none', function() {
-            var message = 'index for write with interval "none" cannot contain *';
-            var index = 'star*';
-            return check_write_fail(test_utils.write([{}], {index: index}), message);
+        it('writes and reads with one index', function() {
+            return check('none');
         });
 
-        it('invalid index name', function() {
-            var time = new Date().toISOString();
-            var write = test_utils.write([{time: time}], {index: 'spaces are bad'})
-                .then(function(result) {
-                    expect(result.errors).match(/Invalid index name/);
-                });
+        describe('errors', function() {
+            function check_write_fail(promise, message) {
+                return check_no_write(expect_to_fail(promise, message));
+            }
+            it('bogus interval', function() {
+                var message = 'invalid interval: bananas; accepted intervals are "day", "week", "month" "year", and "none"';
```

Comment:
the message is missing a comma, is that how it is in the emitted error message too?